libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -164,6 +174,71 @@ void IRGenerator::generateQueuedFunctions()
 		generateFunction(*m_context.dequeueFunctionForCodeGeneration());
 }
 
+InternalDispatchMap IRGenerator::generateInternalDispatchFunctions()
+{
+	solAssert(
+		m_context.functionGenerationQueueEmpty(),
+		"At this point all the enqueued functions should have been generated. "
+		"Otherwise the dispatch may be incomplete."
+	);
+
+	InternalDispatchMap internalDispatchMap = m_context.consumeInternalDispatchMap();
+	for (YulArity const& arity: internalDispatchMap | boost::adaptors::map_keys)
+	{
+		string funName = IRNames::internalDispatch(arity);
+		m_context.functionCollector().createFunction(funName, [&]() {
+			Whiskers templ(R"(
+				function <functionName>(fun <comma> <in>) <arrow> <out> {
```

Comment:
```suggestion
				function <functionName>(fun<?+in>, <in></+in>) <?+out>-> <out> </+out>{
```

Reply:
This is something that's already in `develop` and this PR does not change it (only moves the function) so I fixed it in a separate PR: #9026.